core/src/main/java/org/apache/iceberg/MetricsUtil.java

```diff
@@ -56,4 +64,116 @@ public static MetricsModes.MetricsMode metricsMode(
     String columnName = inputSchema.findColumnName(fieldId);
     return metricsConfig.columnMode(columnName);
   }
+
+  /**
+   * Return a readable metrics map
+   *
+   * @param schema schema of original data table
+   * @param contentFile content file with metrics
+   * @return map of readable column name to column metric, of which the bounds are made readable
+   */
+  public static Map<String, StructLike> readableMetricsMap(
+      Schema schema, ContentFile<?> contentFile) {
```

Comment:
nit: If you call it `file` instead of `contentFile`, the arg definition would fit on one line.